src/views/Search.vue

```diff
@@ -32,7 +32,7 @@ import {
   IonInfiniteScroll,
   IonInfiniteScrollContent,
   IonList,
-  IonListHeader
+  IonListHeader,
```

Comment:
Remove the unwanted comma

Reply:
Removed Unwanted commas.
Thank you Sir for reviewing my code.